### PR TITLE
Update OWB Payment Gateway structure

### PR DESCRIPTION
**Pull request by kloon**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
This PR implements the changes as per p6q8Tx-VJ-p2

### CODE REVIEW

**Review by rodrigoprimo — COMMENTED** [review]
I left some in-line comments with some suggestions. Besides that, I believe that PayPal should be checked for countries where Stripe is not supported, and this is not currently happening. Brazil is an example of such a country.

I also added a commit to this branch to update a unit test that was failing as it was expecting Stripe to be disabled and also two other commits to fix issues in the modified test class (PHPCS errors and wrong usage of `assertEquals()`).

File: assets/css/wc-setup.scss
Comment (kloon) [reply assets/css/wc-setup.scss]: Sorry about that, my editor seems to have auto formated that.

File: includes/admin/class-wc-admin-setup-wizard.php
Comment (kloon) [reply includes/admin/class-wc-admin-setup-wizard.php]: I actually thought about redoing the whole logic there, but since this is a fix release I wanted to keep the changes minimal as to avoid a possible logic error. We can revisit this in 3.7 also with a refactor. IMO we should avoid doing refactors in fix releases.

File: includes/admin/class-wc-admin-setup-wizard.php
Comment (rodrigoprimo) [reply includes/admin/class-wc-admin-setup-wizard.php]: That is a fair point. I don't know if we have a clear policy around what to do in this case. To err on the side of caution, I agree it is probably better to leave this to 3.7.

File: includes/admin/class-wc-admin-setup-wizard.php
Comment (rodrigoprimo) [reply includes/admin/class-wc-admin-setup-wizard.php]: Fair enough. I will create a PR with this change and the removal of the images and tag it for 3.7.

**Comment by rodrigoprimo** [comment]
> Besides that, I believe that PayPal should be checked for countries where Stripe is not supported, and this is not currently happening. Brazil is an example of such a country.

@kloon I believe we still need to address the issue above before merging this PR?

**Comment by kloon** [comment]
@rodrigoprimo OK address that, also had to fix an error in the part where it detected previously saved settings but without a specified option.

**Comment by peterfabian** [comment]
Just tried:
- UK, not selling in person -> Stripe and PayPal appear both, but PP is checked. This seems incorrect, PP should be off?
Screenshot 2019-05-16 at 15 53 03

- UK, selling products in person -> Stripe, Square and PayPal appear, but PP is checked and Square logo should be on white bg?
Screenshot 2019-05-16 at 15 53 29

- Denmark, not selling in person -> all checked, PP shouldn't be, Klarna should be on white background?
Screenshot 2019-05-16 at 15 54 41

- Poland -> PayPal checked, no Stripe or anything else, this seems ok.
Screenshot 2019-05-16 at 15 57 03

**Review by peterfabian — APPROVED** [review]
Seems to work fine now. 👍

**Pull request merged**
